x-pack/plugins/observability_solution/infra/server/lib/alerting/log_threshold/log_threshold_executor.ts

```diff
@@ -169,11 +170,21 @@ export const createLogThresholdExecutor =
             alertDetailsUrl: getAlertDetailsUrl(libs.basePath, spaceId, uuid),
           };
 
+          const instances = alertInstanceId.split(',');
+          const groups =
+            alertInstanceId !== '*'
+              ? params.groupBy?.reduce((resultGroups: any, groupByItem, index) => {
```

Comment:
I need to type it as `any` here because using `{field: string, value: string}[]` - which is the actual type - gives type mismatch error when passing `groups` to `[ALERT_GROUP]`. It expects `field` and `value` to be an array of string.

@maryam-saeidi I see that in custom threshold rule executor, similar logic is implemented but it works without `any` there, do you have any suggestion on how to avoid using `any` here?

Reply:
Thanks @maryam-saeidi, I updated the code in 517b54212a936aed703bb046a58a378aa3b5ae27 as per your suggestion, it worked!